docs/network/migration-guide.md

```diff
@@ -43,11 +43,118 @@ We have an example script here that will perform the re-minting for you: https:/
 
 ## Performing re-encryption
 
-Re-encryption is simply, decrypting the content, then encrypting it again.  In the case of a migration from an old Lit Network to Habanero, you would connect to the old network, decrypt the user’s data, and then connect to Habanero and encrypt it again.
+Re-encryption is simply, decrypting the content, then encrypting it again. The v3 SDK of the encryption system introduces significant enhancements compared to v2 SDK. It employs a more intricate yet secure method of encryption and decryption, utilizing hashes and a comprehensive set of parameters to bolster security and integrity. 
```

Comment:
Add about ID based encryption and that the encryption part happens completely client-side: https://developer.litprotocol.com/v3/sdk/access-control/encryption#how-does-id-encrypt-work

Reply:
Done @Adarsh-Kumar28 :)